src/BuildScriptGenerator/Node/NodeConstants.cs

```diff
@@ -5,26 +5,30 @@
 
 namespace Microsoft.Oryx.BuildScriptGenerator.Node
 {
-    internal static class NodeConstants
+    public static class NodeConstants
```

Comment:
For now I made this public, but we can revisit this when we plan to release this as a library and do not want to expose this. In that case we can do the `InternalsVisibleTo`

Reply:
where is it used?

Reply:
We should already have an internalsVisibleTo the tests... I'd do that now rather than later, since once it is public we might start relying on it and fixing it later will be more expensive

Reply:
Sure, will change it